### PR TITLE
Add mne-gui-addons

### PR DESCRIPTION
**Pull request by larsoner**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/mne-gui-addons/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+about:
+  home: https://pypi.org/project/mne-gui-addons/
+  summary: MNE-Python GUI addons.
+  license: BSD-3-Clause
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - larsoner
+    - alexrockhill
```
Comment (larsoner) [inline recipes/mne-gui-addons/meta.yaml:55]: @alexrockhill can you confirm that you're willing to co-maintain this recipe?

Comment (alexrockhill) [reply recipes/mne-gui-addons/meta.yaml:55]: Yes

**Comment by larsoner** [comment]
https://github.com/orgs/conda-forge/teams/help-python this is ready for review, thanks!

The Window failure seems to be due to `import pyvista` failing due to `import vtk` failing on Windows. I'm assuming that this will be "fixed" because of this being a "noarch" package as well so in the end only the Linux build will matter. But if that's not the case then I'd need help debugging/fixing the failures :(

**Comment by larsoner** [comment]
@conda-forge/help-python this is ready for review, thanks!

The Window failure seems to be due to import pyvista failing due to import vtk failing on Windows. I'm assuming that this will be "fixed" because of this being a "noarch" package as well so in the end only the Linux build will matter. But if that's not the case then I'd need help debugging/fixing the failures :(

**Comment by larsoner** [comment]
@conda-forge/help-python any chance to get this merged despite the windows failure given that this is `noarch` and the Windows failure seems to be an upstream VTK problem?

**Comment by hoechenberger** [comment]
Thank you, @wolfv!!

**Pull request merged**